src/components/test/input.ts

```diff
@@ -148,9 +153,14 @@ const properties: Properties = {
             type: 'integer'
         }
     },
+    gps: {
+        widget: {
+            type: 'gps'
+        }
+    },
     image: {
         widget: {
-            type: 'image'
+            type: 'imageUpload'
```

Comment:
Can we keep type: image and add another property for imageUpload ? This will probably also simplify the changes in the snapshots.